lib/icedfrisby.js

```diff
@@ -125,324 +127,322 @@ function _useAppImpl (app, basePath) {
-  return this;
-};
+  /**
+   * @param {string} app - Node.js app. (e.g. Express, Koa)
+   * @param {string} basePath - base path to append to the server address
+   * @return {object}
+   * @desc Setup the request to use a passed in Node http.Server app
+   */
+  useApp (app, basePath) {
+    this.current.request.baseUri = _useAppImpl(app, basePath);
+    return this;
+  }
 
 
-/**
- * @param {number} ms - timeout value in milliseconds
- * @return {object}
- * @desc Timeout getter and setter
- */
-Frisby.prototype.timeout = function(ms) {
-  if(!ms) {
-    return this._timeout;
+  /**
+   * @param {number} ms - timeout value in milliseconds
+   * @return {object}
+   * @desc Timeout getter and setter
+   */
+  timeout (ms) {
+    if(!ms) {
+      return this._timeout;
+    }
+    this._timeout = ms;
+    return this;
   }
-  this._timeout = ms;
-  return this;
-};
 
 
-/**
- * @return {object}
- * @desc Reset Frisby global and setup options
- */
-Frisby.prototype.reset = function() {
-  this.current.request = _.cloneDeep(_icedfrisbyDefaults().request);
-  return this;
-};
+  /**
+   * @return {object}
+   * @desc Reset Frisby global and setup options
+   */
+  reset () {
+    this.current.request = _.cloneDeep(_icedfrisbyDefaults().request);
+    return this;
+  }
 
 
-/**
- * @param {string} header - header key
- * @param {string} content - header value content
- * @return {object}
- * @desc Add HTTP header by key and value
- */
-Frisby.prototype.not = function() {
-  this.current.isNot = true;
-  return this;
-};
+  /**
+   * @param {string} header - header key
+   * @param {string} content - header value content
+   * @return {object}
+   * @desc Add HTTP header by key and value
+   */
+  not () {
+    this.current.isNot = true;
+    return this;
+  }
 
 
-/**
- * @param {string} header - header key
- * @param {string} content - header value content
- * @return {object}
- * @desc Add HTTP header by key and value
- */
-Frisby.prototype.addHeader = function(header, content) {
-  this.current.request.headers[(header+"").toLowerCase()] = content+"";
-  return this;
-};
+  /**
+   * @param {string} header - header key
+   * @param {string} content - header value content
+   * @return {object}
+   * @desc Add HTTP header by key and value
+   */
+  addHeader (header, content) {
+    this.current.request.headers[(header+"").toLowerCase()] = content+"";
+    return this;
+  }
 
 
-/**
- * @param {object} headers - header object {k;v, k:v}
- * @return {object}
- * @desc Add group of HTTP headers together
- */
-Frisby.prototype.addHeaders = function(headers) {
-  const self = this;
-  _.forEach(headers, function(val, key) {
-    self.addHeader(key, val);
-  });
-  return this;
-};
+  /**
+   * @param {object} headers - header object {k;v, k:v}
+   * @return {object}
+   * @desc Add group of HTTP headers together
+   */
+  addHeaders (headers) {
+    const self = this;
+    _.forEach(headers, function(val, key) {
```

Comment:
Yes, that would be great! They'll make these things much more readable.

Since those changes are easy to get wrong, having more eyes to review them is good. It's impossible to review the diff here, so I would tend to make those in a separate commit.